dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/LDAPRegistryContextPoolNoConfigTest.java

```diff
@@ -149,6 +149,10 @@ public void testCustomContextNoConfig() throws Exception { // for issue
         tr = "CacheResultSizeLimit: 2000";
         errMsgs = server.findStringsInLogsAndTrace(tr);
         assertFalse("Should have found, " + tr, errMsgs.isEmpty());
+
```

Comment:
I don't think this belongs here. We probably want to put it in its own FAT or something more closely related. This FAT deals with the context pool. Additionally alls this tests is whether some trace is printed out and not that it mirrors the value passed in to the read timeout.

I would write a test that tries to call out to an LDAP server that is unavailable and verify that you get a connection timeout after 60 seconds. I would then modify the the read timeout dynamically to something short and verify that again you get a connection timeout, but this time much quicker.

Reply:
Actually, this is read timeout, so you might have to just open your own connection and never respond to LDAP requests. That should get you a read timeout I imagine.